### DISCUSSION TITLE
Getting errors with eth_call method and eth_getBalance method ?

### DISCUSSION THREAD

**Discussion by doubleErr** [WalletConnect/walletconnect-monorepo#2809] [Developer Support]
Although i updated both etherjs and wagmi like that:

"wagmi": "^0.12.16",
"ethers": "^5.7.2"

i still got this issue
<img width="890" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/117885541/636e8793-239f-4848-89fb-854d91d58219">

**Comment by finessevanes** [comment]
Hey could you share a minimal reproducible example?

**Reply by doubleErr** [reply]
thank for your response, i ll reproduce and send u soon

**Reply by doubleErr** [reply]
hmm seems like our dapp is working properly again without changing anything :-? seems like wagmi'api just updated right ?

**Reply by doubleErr** [reply]
btw we are running on goerli

**Reply by doubleErr** [reply]
i think you guys just fixed some issues on wagmi server right @finessevanes

**Reply by finessevanes** [reply]
@doubleErr did you update your wagmi package?

**Reply by doubleErr** [reply]
nah @finessevanes , i didnt change anything, but errors were gone, it happened in 2 days but now everything is oke, i just want to confirm if something changes in wagmi server
btw its only happened in goerli, mainnet is still working properly
